hrm-service/src/app.js

```diff
@@ -137,10 +137,11 @@ app.use((err, req, res, next) => {
 
   // set locals, only providing error in development
   res.locals.message = err.message;
-  res.locals.error = req.app.get('env') === 'development' ? err : {};
+  const env = req.app.get('env');
```

Comment:
This should be controlled by it's own config item rather than being driven off the env, at least primarily, setting based on env can be a fallback (we need to have a set of default flags for each env that can be overridden individually going forward)